actions/run_action.go

```diff
@@ -98,8 +98,8 @@ func (run *RunAction) doRun(context debos.DebosContext) error {
 		script[0] = debos.CleanPathAt(script[0], context.RecipeDir)
 		if run.Chroot {
 			scriptpath := path.Dir(script[0])
-			cmd.AddBindMount(scriptpath, "/script")
-			script[0] = strings.Replace(script[0], scriptpath, "/script", 1)
+			cmd.AddBindMount(scriptpath, "/tmp/script")
+			script[0] = strings.Replace(script[0], scriptpath, "/tmp/script", 1)
```

Comment:
This does assume the /tmp in the chroot is on a tmpfs otherwise it'll still leave it around (though in a less awkward location);  Is this guaranteed ? (Otherwise making the cmd functionality explicitly clean up afterwards would be nicer)

Reply:
This change is related for `systemd-nspawn` only.
And there is [a commit](https://github.com/systemd/systemd/commit/bbb99c30d01a8bcdc27fb151cc6376a7877a6b07) which explicitly mount `tmpfs` to `/tmp` for container.

Reply:
Could you note that in the commit message or a comment please? otherwise looks fine

Reply:
Done